src/index.js

```diff
@@ -113,47 +113,53 @@ class Header {
   renderSettings() {
     let holder = document.createElement('DIV');
 
+    // do not add settings button, when only one level is configured
+    if (this.levels.length <= 1) {
+      return holder;
+    }
+
     /** Add type selectors */
-    this.levels.forEach( level => {
+    this.levels.forEach(level => {
       let selectTypeButton = document.createElement('SPAN');
 
       selectTypeButton.classList.add(this._CSS.settingsButton);
 
       /**
-       * Highlight current level button
-       */
+      * Highlight current level button
+      */
       if (this.currentLevel.number === level.number) {
         selectTypeButton.classList.add(this._CSS.settingsButtonActive);
       }
 
       /**
-       * Add SVG icon
-       */
+      * Add SVG icon
```

Comment:
Something happened with jsdoc comments indentation.

Reply:
Yeah, my prettier happend ;) Gonna change it.